webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -1065,24 +1065,10 @@ object RudderConfig extends Loggable {
   // Plugin input interface for Authorization for API
   lazy val authorizationApiMapping = new ExtensibleAuthorizationApiMapping(AuthorizationApiMapping.Core :: Nil)
 
-  ////////// end plugable service providers //////////
+  ////////// end pluggable service providers //////////
 
   lazy val roleApiMapping = new RoleApiMapping(authorizationApiMapping)
 
-  lazy val rudderUsernameCaseSensitive: Boolean = {
-    (for {
```

Comment:
It is dangerous to define that as a global parameter while the underlying file providing it can be reloaded. In the best case, we end up with a desynchonisation between that property and the one in the file making debugging harder, in the worst case we actually don't do what the user asked in the config file and make a complicated WTF user experience where things only work when restarting rudder.